src/Compilers/CSharp/Test/Emit/CSharpCompilerEmitTest.csproj

```diff
@@ -55,9 +56,6 @@
     <Reference Include="..\..\..\..\..\packages\System.Reflection.Metadata.$(SystemReflectionMetadataVersion)\lib\portable-net45+win8\System.Reflection.Metadata.dll" />
     <Reference Include="..\..\..\..\..\packages\System.Collections.Immutable.$(SystemCollectionsImmutableVersion)\lib\portable-net45+win8+wp8+wpa81\System.Collections.Immutable.dll" />
     <Reference Include="..\..\..\..\..\packages\Microsoft.DiaSymReader.1.0.5\lib\net45\Microsoft.DiaSymReader.dll" />
-    <Reference Include="xunit">
-      <HintPath>..\..\..\..\..\packages\xunit.1.9.2\lib\net20\xunit.dll</HintPath>
-    </Reference>
```

Comment:
@KevinH-MS it's either 

1. Add a flag to every test project
2. Add an `Import`

I'd rather add the flag.  It's much more direct and we already have a precedence of letting flags in the project file control how we process items in the targets files.